lib/data/locationConstraintParser.js

```diff
@@ -72,6 +73,23 @@ function parseLC() {
             });
             clients[location].clientType = 'aws_s3';
         }
+        if (locationObj.type === 'azure') {
+            // AZURE_BLOB_ENDPOINT and AZURE_BLOB_SAS environement variables
+            // have been added for our internals tests
+            const azureBlobEndpoint = process.env.AZURE_BLOB_ENDPOINT ?
```

Comment:
hmmm...let's make the environment variable the actual locationConstraint name followed by azure blob endpoint so that there aren't unintentional overrides and so you could potentially use env variables for all of them, so:
const envName = `${location}_AZURE_BLOB_ENDPOINT
const azureBlobEndpoint = process.env[envName] ? ...